src/cpp.hint

```diff
@@ -0,0 +1,19 @@
+// This file is needed to workaround some of the buggy intellisense
+#define API __declspec(dllexport)
+#define API __declspec(dllimport)
```

Comment:
why define twice with two different values?

Reply:
every macro in this file is double defined except `_NODISCARD`.

Reply:
This file isn't needed anymore